project.md

```diff
@@ -1,33 +1,24 @@
-# Project Idea Pitch
+An app or website for teenagers to make money.
 
-Edit this file to include your project idea pitch for the HighTechU Academy.
+Teenagers want to make money, but it's hard to find a job while also being a full time student. This app would let people post work shifts with flexible hours as well as odd jobs, both physical and online.
 
 ## Project Title
 
-[DELETE ME - ADD YOUR PROJECT TITLE HERE]
-
-Example - Colour Picker
+I haven't thought of a  good title for this yet.
 
 ## Project Mini Pitch
 
-[DELETE ME - ADD YOUR PROJECT MINI PITCH HERE]
-
-Example - This colour picker will revolutionaize the way colours are picked for the web.
+This app would provide a way for busy teenagers to make some extra cash without completely filling up their schedule.
 
 ## Project Description
 
-[DELETE ME - ADD YOUR PROJECT DESCRIPTION HERE]
-
-Example - The app will be web based and allow users to pick a colour with the help of a colour wheel.
+It will be an app for teenagers who are busy with school to make money on the side. They can take up odd jobs, as well as flexible shifts. Both physical and online jobs could be posted, for example if someone was making a personal website and wanted a cool banner at the top, they could make a post about wanting one, and if someone was interested they could go into further detail.
 
 ## Project Audience
 
-[DELETE ME - ADD YOUR PROJECT AUDIENCE HERE]
-
-Example - The audience would be frontend developers and designers looking to pick colours for their work.
+The main audience of this app will be teenagers looking for a way to make some money while still being a full-time student.
 
 ## Project Inspiration
 
-[DELETE ME - ADD YOUR PROJECT INSPIRATION HERE]
+I didn't have a bunch of inspiration for this, but the app Fiverr gave me the idea for the online aspect of the odd jobs catagory. It also seems like something that I would use if it was available, which helped me to come up with some of the ideas.
 
-Example - This project idea was inspired by the following websites: list of website.
```

Comment:
Hey Blaise!

Thanks for submitting this project idea! It sounds super interesting. A job board for teens is a great idea.